storage/src/context_action_storage.rs

```diff
@@ -476,7 +874,6 @@ mod tests {
 
     use super::*;
 
-    #[test]
```

Comment:
@Kyras 
What about this removed #[test]? 
You should also remove test function, if there is no need, or implement test correctly pls. ;)

Reply:
Those tests, were not correct, as the associated RPC filters did not worked correctly, and yes they should be remade to reflect the new changes.